jpa/hibernate6/src/main/java/org/jboss/as/jpa/hibernate/management/HibernateManagementAdaptor.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.as.jpa.hibernate.management;
+
+import org.jboss.as.jpa.spi.ManagementAdaptor;
+import org.jipijapa.management.spi.Statistics;
+
+/**
+ * Contains management support for Hibernate
+ *
+ * @author Scott Marlow
+ */
+public class HibernateManagementAdaptor implements ManagementAdaptor {
+
+    // shared (per classloader) instance for all Hibernate 4.3 Jakarta Persistence deployments
+    private static final HibernateManagementAdaptor INSTANCE = new HibernateManagementAdaptor();
```

Comment:
Maybe it's intended: A singleton with an static instance but the constructor is default and therefor public. Is a private constructor needed here?

Reply:
Thanks @boris-unckel , just switched to a private constructor